src/main/storm/mesos/MesosNimbus.java

```diff
@@ -152,16 +153,21 @@ protected void initialize(Map conf, String localDir) throws Exception {
     }
     _container = Optional.fromNullable((String) conf.get(CONF_MESOS_CONTAINER_DOCKER_IMAGE));
     _scheduler = new NimbusScheduler(this);
+
+    // Generate YAML to be served up to clients
+    _generatedConfPath = Paths.get(
+        Optional.fromNullable((String) conf.get(Config.STORM_LOCAL_DIR)).or("./"),
+        "generated-conf");
+    if (!_generatedConfPath.toFile().exists() && !_generatedConfPath.toFile().mkdirs()) {
+      throw new RuntimeException("Couldn't create generated-conf dir");
```

Comment:
Maybe add in the path that couldn't be created to the exception?  ` _generatedConfPath`